src/Utils/createStripeWrapper.tsx

```diff
@@ -1,13 +1,13 @@
-import { loadStripe } from '@stripe/stripe-js';
+import { loadStripe, StripeElementsOptions } from '@stripe/stripe-js';
 import { Elements, ElementsConsumer } from "@stripe/react-stripe-js"
 import * as React from "react";
 import { data as sd } from "sharify"
 
-export function createStripeWrapper<T>(Component: React.FC<T>): React.FC<T> {
+export function createStripeWrapper<T>(Component: React.FC<T>, stripeElementsOptions?: StripeElementsOptions): React.FC<T> {
```

Comment:
Expanding the wrapper to take an additional optional argument to be passed as `options` in Stripe `Elements`.